messages/compatibility.json

```diff
@@ -1,5 +1,5 @@
 {
   "sourceTrackingFileVersionMismatch": "This project uses the %s version of source tracking files.",
   "clearSuggestion": "Clear the %s version of the tracking files by running '%s'",
-  "useOtherVersion": "Use the %s version of the command, '%s' to preserve the tracking files"
+  "useOtherVersion": "Use the %s version of the command, '%s' with your existing tracking files"
```

Comment:
I didn't like the way this worked with reset/clear since you AREN'T preserving the tracking files in those cases.

Reply:
```suggestion
  "useOtherVersion": "Use the %s version of the command, '%s' with your existing tracking files."
```